projects/mysadna/src/app/models/contribution.ts

```diff
@@ -1,7 +1,7 @@
 export class Contribution {
-    contributionId: number;
-    userId: number;
-    projectId: number;
-    contributionMessage: string;
-    contributionUrl: string;
+  contributionId: number;
```

Comment:
2 space indent size

Reply:
SERIOUSLY?! Why can't I just define 2 spaces as the default indent size for **all** my projects in VS Code?!

Reply:
Yeah.. Maybe there's a plugin for this?